README.md

```diff
@@ -1,17 +1,21 @@
 # Open Container Specifications
 
-This project is where the [Open Container Initiative](http://www.opencontainers.org/) Specifications are written.
-This is a work in progress.
+[Open Container Initiative](http://www.opencontainers.org/) Specifications for standards on Operating System process and application containers.
```

Comment:
On Fri, Sep 25, 2015 at 08:15:03AM -0700, Vincent Batts wrote:
> > -This project is where the [Open Container Initiative](http://www.opencontainers.org/) Specifications are written.
> > -This is a work in progress.
> > +[Open Container Initiative](http://www.opencontainers.org/) Specifications for standards on Operating System process and application containers.
> 
> … Neither of these things has to do with operating systems.

How about:

> [Open Container Initiative](http://www.opencontainers.org/) Specifications for application containers.
> A framework for defining containers on several operating systems and containerization technologies.

To cover both Linux / Windows / … and Linux-containers /
Windows-containers / hypervisor-virtual-hosts/….